packages/components/src/components/hds/rich-tooltip/toggle.hbs

```diff
@@ -17,13 +17,13 @@
     {{yield}}
   {{~else~}}
     {{~#if (and @icon (eq this.iconPosition "leading"))~}}
-      <FlightIcon class="hds-rich-tooltip__toggle-icon" @name={{@icon}} @isInlineBlock={{this.isInline}} />
+      <Hds::Icon class="hds-rich-tooltip__toggle-icon" @name={{@icon}} />
     {{~/if~}}
     {{~#if @text~}}
       <span class="hds-rich-tooltip__toggle-text">{{~@text~}}</span>
     {{~/if~}}
     {{~#if (and @icon (eq this.iconPosition "trailing"))~}}
-      <FlightIcon class="hds-rich-tooltip__toggle-icon" @name={{@icon}} @isInlineBlock={{this.isInline}} />
```

Comment:
The codemod doesn't appear to work on dynamic values for `@isInlineBlock`

Reply:
What would you expect the codemod to do in this case?

Reply:
It was removing the attribute, I have since updated the codemod so that it will update correctly. I also added another test case for this scenario.

Reply:
Ahh, ok. Now I understand :)

Reply:
Quick question: since you've found this use case that was not covered, what do you think to do a dry run test on Cloud UI to see if other use cases are not covered? (I don't think there are, but just to be on the safe side)